### DISCUSSION TITLE
A question about Github copilot student dev pack

### DISCUSSION THREAD

**Discussion by 0niDev** [community/community#61007] [GitHub Education]
### Hi everyone,

As a diligent and enthusiastic college student at Roots Ivy International, I was delighted to receive an email account associated with the college domain - "S-6-********[.fsd@rootsivy.edu.pk](mailto:.fsd@rootsivy.edu.pk)." This email comes with exciting possibilities, including access to a Google Teams account that the college generously allowed me to use.

But when I tried to apply for Github copilot student dev pack they wanted my to verify the email but I don't have the password for the email is there a way i can apply for the for Github copilot student pack without verifying this email

Thank you for your attention to this matter, and I look forward to your prompt response and resolution.

**Comment by samus-aran** [comment]
Hi @OniChan-Baka 

To be eligible for GitHub Global Campus, including GitHub Student Developer Pack and other benefits, you must:

- Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution
- Have a **verifiable school-issued email address** or upload documents that prove your current student status
- Have a [GitHub personal account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account)
- Be at least 13 years old

You can use alternative pieces of evidence but in this case it would be the Academic email(s) which has been associated with this school. We would recommend to you speak to your institution regarding to acquiring a password to this email address to proceed.

**Reply by 0niDev** [reply]
Thanks for the suggestion I did ask them about the password but they just left me on read. I will try to contact them again.
